api/v1alpha1/virtualmachineclassbinding_types.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2020 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ClassReference contains info to locate a specific VirtualMachineClass object
+type ClassReference struct {
+	// API version of the referent.
+	APIVersion string `json:"apiVersion,omitempty"`
+	// Kind is the type of resource being referenced.
+	Kind string `json:"kind"`
+	// Name is the name of resource being referenced.
```

Comment:
What's the value in specifying the `APIVersion` and `Kind`? The referenced object will always be a VirtualMachiineClass, and a name is unique enough (it will be unique across APIVersions as well). 

I think just `Name` here is sufficient?

Reply:
Yeah. I think `Kind` is redundant atm. APIVersion with an omitempty and Name should suffice.